react-app-rewired/headers/csps/defi/osmosis.ts

```diff
@@ -0,0 +1,8 @@
+import type { Csp } from '../../types'
+
+export const csp: Csp = {
+  'connect-src': [
+   'https://lcd-osmosis.keplr.app/osmosis/',
```

Comment:
these can be updated to use the env vars from `config.ts` for a single source of truth - lots of examples how to do this

Reply:
Ah yes, this was a temporary hack that I forgot to revert. The environment variables wouldn't resolve properly, but I realized later that I needed to 'yarn env dev' after adding the new env vars to the config.